specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-01-14-preview/examples/SessionHost_Delete.json

```diff
@@ -26,7 +26,18 @@
         "lastUpdateTime": "2008-09-22T14:01:54.9571247Z",
         "updateErrorMessage": "",
         "virtualMachineId": "29491b54-c033-4dec-b09a-18bf0ebafaef",
-        "resourceId": "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourceGroups/resourceGroup1/providers/Microsoft.Compute/virtualMachines/sessionHost1"
+        "resourceId": "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourceGroups/resourceGroup1/providers/Microsoft.Compute/virtualMachines/sessionHost1",
+        "sessionHostHealthCheckResults": [
+          {
+            "healthCheckName": "0",
+            "healthCheckResult": "1",
+            "additionalFailureDetails": {
+              "message": "SessionHost healthy: is joined to domain ≤wvdarmtest1.net≥",
+              "errorCode": 0,
+              "lastHealthCheckInUTC": "2021-01-14T02:09:37.6236843Z"
```

Comment:
Once you make the property name change, make sure to update your examples as well.

Reply:
all updated